src/app/docs/architecture-design/performance/page.tsx

```
@@ -12,71 +11,135 @@ import {
       <div className="mb-12">
-        <h1 className="text-4xl font-bold text-gray-900 mb-6">性能详解</h1>
-        <p className="text-xl text-gray-600 leading-relaxed">
-          GPT-Load 采用<strong>&ldquo;代理路径优先&rdquo;</strong>
-          的高性能设计哲学， 一切优化都为保障核心代理请求的极致性能和稳定性。
-        </p>
+        <h1 className="text-4xl font-bold text-gray-900 mb-6">
+          {t("header.title")}
+        </h1>
+        <p
+          className="text-xl text-gray-600 leading-relaxed"
+          dangerouslySetInnerHTML={{ __html: t.raw("header.description") }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)